src/main/resources/liquibase/db.changelog-1.1.xml

```diff
@@ -83,43 +77,105 @@
                 <constraints nullable="true"/>
             </column>
 
-            <column name="created" type="TIMESTAMP" defaultValue="CURRENT_TIMESTAMP()">
-                <constraints nullable="false"/>
-            </column>
-
-            <column name="delivered" type="BOOLEAN" defaultValue="false">
+            <column name="status" type="VARCHAR(7)" defaultValue="false">
                 <constraints nullable="true"/>
             </column>
 
-            <column name="modified" type="BOOLEAN" defaultValue="false">
-                <constraints nullable="true"/>
+            <column name="created" type="TIMESTAMP" defaultValue="CURRENT_TIMESTAMP()">
+                <constraints nullable="false"/>
             </column>
 
-            <column name="was_read" type="BOOLEAN" defaultValue="false">
-                <constraints nullable="true"/>
+            <column name="updated" type="TIMESTAMP" defaultValue="CURRENT_TIMESTAMP()">
+                <constraints nullable="false"/>
             </column>
         </createTable>
     </changeSet>
 
-    <changeSet id="1.1.5" author="jack.petrov">
+    <changeSet id="1.1.6: Create a foreign key between 'message' and 'chat' tables (many-to-one relationship)" author="jack.petrov">
         <addForeignKeyConstraint baseColumnNames="chat_id"
                                  baseTableName="message"
                                  constraintName="fk_message_to_chat_id"
                                  onDelete="CASCADE"
                                  onUpdate="RESTRICT"
                                  referencedColumnNames="id"
-                                 referencedTableName="chat_room"/>
+                                 referencedTableName="chat"/>
     </changeSet>
 
-    <changeSet id="1.1.6" author="jack.petrov">
+    <changeSet id="1.1.7: Create a foreign key between 'message' and 'user' tables (many-to-one relationship)" author="jack.petrov">
         <addForeignKeyConstraint baseColumnNames="user_id"
-                                 baseTableName="messages"
+                                 baseTableName="message"
                                  constraintName="fk_message_to_user_id"
                                  referencedColumnNames="id"
                                  referencedTableName="user"/>
     </changeSet>
+
+    <changeSet id="1.1.8: Create a table 'attachment'" author="jack.petrov">
+        <createTable tableName="attachment">
+            <column name="id" type="BIGINT" autoIncrement="true">
+                <constraints primaryKey="true" unique="true" nullable="false"/>
+            </column>
+
+            <column name="message_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="type" type="VARCHAR(7)">
+                <constraints nullable="true"/>
+            </column>
+
+            <column name="file_content" type="BLOB">
+                <constraints nullable="true"/>
```

Comment:
Set nullable to true here and in 'type' value as well. I don't think we going to create attachment without attachment.

Reply:
Done